<?xml version="1.0"?>
<launch>
<arg name="enable_mock" default="False"/>
<arg name="robot_namespace" default="/"/>
<arg name="verbosity_level" default="DEBUG"/> 
<node name="sonar_array_driver_node" pkg="sonar_array" type="sonar_array_driver_node"  output="screen" clear_params="true">
        <param name="enable_mock"               value="$(arg enable_mock)"/>
        <param name="robot_namespace"           value="$(arg robot_namespace)"/>
        <param name="startup_delay"             value="0.0"/>
	<param name="verbosity_level"           value="$(arg verbosity_level)"/>
        <param name="require_pps_to_start"      value="false"/>    
        <param name="loop1_rate"                value="50"/> 
        <param name="loop2_rate"                value="10"/>
        <param name="loop3_rate"                value="5"/>
</node>
</launch>
